GalnetMonitor/Galnet list news headlines.txt

```diff
@@ -0,0 +1,36 @@
+{_ Fetch article category and boolean include read articles from state }
+{set spokenmonth to ["January","February","March","April","May","June","July","August","September", "October","November","December"]}
+
+{_ SetState("category", "All")}
+{_ SetState("inc_read", "true")}
+
+{set unread to eq(state.inc_read,"true")}
+{set articles to GalnetNewsArticles("All",true)}
+
+{if len(articles) = 0:
+    There is no news
+|else:
+{set lastdate to articles[1].published}
+
+Gal net News
+{Pause(150)}
+Your galaxy in focus
+{Pause(150)}
+These are the headlines for {spokenmonth[lastdate.Month-1]} {lastdate.Day}
+{Pause(800)}
+{set index to 0}
+{for news in articles:
+  {set index to index + 1}
+  {if (ne(news.read,true) || unread) && eq(news.category,state.category):
+    {set item to split(news.title, ":")}
+    item {index} 
+    {Pause(150)}
+    {if ne(news.category,"Article"): {item[0]} {Pause(150)}}
+    {item[1]}
+    {Pause(1000)}
+  }
+}
+End of Stream
+{Pause(500)}
+{OneOf("Gal net News Transmissions are sponsored in part by the Bank of Zaoncee. Trust the Bank of Zaoncee with your hard-earned credits!","This news was brought to you by Delacy! Delacy! From Sidewinder to Anaconda: classic ship design to fullfill every wish!","Fast and powerful! These attributes are true for Gal net News AND for Core Dynamics ships! Gal net News does neither endorse nor condemn federal policies!","This Gal net News Transmission brought to you by Zorgon Peterson When real work needs to be done: Zorgon Peterson!","Laykon! When space is not only where you are, but is also what you need! Proudly sponsoring this newscast for you!","Saud Kruger sponsored this newsfeed. Fly an Orca and show the world that elegance and beauty trumps it all!","Gutamayas Imperial Clipper! As smoothly reliable as Gal net News. Gal net News does neither endorse nor condemn imperial policies!","Remember to check the Station Service for local news not covered by this feed. Fly safely commanders.")}
+}.
```

Comment:
Since this is an example script only and not intended for incorporation, please remove this file from the pull request.